normalize.js

```diff
@@ -111,11 +126,8 @@ define(function() {
     
     return out.substr(0, out.length - 1);
   };
-  
-  var normalizeCSS = function(source, fromBase, toBase) {
 
-    fromBase = removeDoubleSlashes(fromBase);
-    toBase = removeDoubleSlashes(toBase);
+  var normalizeCSS = function(source, fromBase, toBase) {
```

Comment:
The function `relativeURI` has already ended at line 128, `var normalizeCSS = function () {` is at the same indent level as `relativeURI`, and line 132, 133, 135 etc is **inside** the function which make them one more indent level than `var normalizeCSS`.

<img width="985" alt="screen shot 2017-02-20 at 11 15 45 am" src="https://cloud.githubusercontent.com/assets/5248918/23110758/9b35ff1e-f75e-11e6-9848-a3eaf9126b7b.png">